src/StringUtils.h

```diff
@@ -142,9 +142,12 @@ The order of items doesn't change, only the duplicates are removed.
 If a_Strings1 contains duplicates, the result will still contain those duplicates. */
 extern AStringVector MergeStringVectors(const AStringVector & a_Strings1, const AStringVector & a_Strings2);
 
-/** Concatenates the specified strings into a single string, separated by the specified separator. */
+/** Concatenates the specified strings into a single string, separated by the specified separator character. */
 extern AString StringsConcat(const AStringVector & a_Strings, char a_Separator);
 
+/** Concatenates the specified strings into a single string, separated by the specified separator string. */
+extern AString StringsConcat(const AStringVector & a_Strings, const AString & a_Separator);
```

Comment:
This already exists, it's called `StringJoin`.

Reply:
Discoverability almost zero. I'll fix this. Thx.